addons/sale/models/res_company.py

```diff
@@ -10,6 +10,7 @@ class ResCompany(models.Model):
     portal_confirmation_sign = fields.Boolean(string='Online Signature')
     portal_confirmation_pay = fields.Boolean(string='Online Payment')
     quotation_validity_days = fields.Integer(default=30, string="Default Quotation Validity (Days)")
+    quotation_confirmation_percentage = fields.Float(string="Requested pay down percentage for quotation confirmation", default=100.0)
```

Comment:
Maybe add an SQL constraint to force the percentage to be in [0,100]

Reply:
That way you won't need the sql constraint 'confirmation_percentage' in sale.py.